CONTRIBUTING.md

```diff
@@ -0,0 +1,26 @@
+## Quickstart
+
+To install and build `quadbin-js` locally from source:
+
+```bash
+# install dependencies
+yarn
+
+# build package once
+yarn build
+```
+
+To run tests, coverage, or a linter, you should execute `yarn build`, and afterward:
+
+```bash
+# run tests once
+yarn test
+```
+
+## Releases
+
+1. Create a new version: `yarn version [ major | minor | patch | prerelease ]`
+
+2. Commit, tag, and push to GitHub: `yarn postversion`
```

Comment:
Note that the `postversion` script from `@carto/api-client` would need to be added to the quadbin.js `package.json`, it is custom:

https://github.com/CartoDB/carto-api-client/blob/f77ec94cf4fff4f89ed5fab2d7cdd9eb6fe52a5f/package.json#L41-L44

Not sure if there's an easier way to do all this in Yarn; Lerna does more but just for monorepos.

Reply:
Wops! Completely forgot about this, my bad 🤦 

I think for simplicity, I'll just add a `prepublishOnly` option in the `package.json`, and will remove this part because this package doesn't seem complex enough to be handled using external tools 👍